src/main/java/roomescape/dto/ReservationResponse.java

```diff
@@ -0,0 +1,21 @@
+package roomescape.dto;
+
+import java.time.LocalDate;
+import roomescape.domain.Reservation;
+
+public record ReservationResponse(Long id,
+                                  String name,
+                                  LocalDate date,
+                                  ReservationTimeResponse time,
+                                  ThemeResponse theme) {
```

Comment:
```suggestion
public record ReservationResponse(
    Long id,
    String name,
    LocalDate date,
    ReservationTimeResponse time,
    ThemeResponse theme
) {
```

이런 부분도 다른 클래스들과 일관성 있게 통일되면 좋을것 같아요!
ReservationTimeResponse랑 다른 형식이라서요

Reply:
모두 후자로 컨벤션을 통일했습니다!